lib/widgets/custom_button.dart

```diff
@@ -4,7 +4,7 @@ import 'package:flutter/material.dart';
 class CustomButton extends StatelessWidget {
   const CustomButton(this.buttonText, {super.key});
 
-  final String buttonText;
+  final String buttonText:
```

Comment:
Please that is a big mistake, you need to use a semi colon to terminate a line of code in flutter.